dataverse-webapp/src/main/resources/Bundle_en.properties

```diff
@@ -1957,6 +1957,21 @@ mydataFragment.roles=Roles
 mydataFragment.resultsByUserName=Results by Username
 mydataFragment.search=Search my data...
 
+#mydata_fragment.xhtml
+Published=Published
+Unpublished=Unpublished
+Draft=Draft
+In_Review=In Review
+Deaccessioned=Deaccessioned
+mydataFragment.cards.publicationStatus=Unpublished TEST
+
+mydataFragment.errorMessage.NoDvObjectsSelected=No results. Please select one of Dataverses, Datasets, Files.
+mydataFragment.errorMessage.NoPublicationStatusSelected=No results. Please select one of
+mydataFragment.errorMessage.NoRoleSelected=No results. Please select at least one Role.
+mydataFragment.errorMessage.NoUserSelected=Sorry!  No user was found!
+mydataFragment.errorMessage.UnknownType.Prefix=Sorry!  The type '
+mydataFragment.errorMessage.UnknownType.Suffix=' is not known.
```

Comment:
Can we stick to one way of casing property keys? Almost all of them are camelCase.

If we mix casing then when searching for usages of property it can confuse developer that it have some special meaning to it (like for example class name or enum value)